packages/cli/lib/lib/webpack/webpack-client-config.js

```diff
@@ -273,9 +260,13 @@ function isDev(config) {
 }
 
 module.exports = function (env) {
-	return merge(
+	const config = [merge(
 		baseConfig(env),
 		clientConfig(env),
 		(env.isProd ? isProd : isDev)(env)
-	);
-};
+	)];
+	if (env.sw) {
```

Comment:
Is the Array return here done with the intention of moving to a MultiCompiler?  This seems like it'll break in future Workbox versions that use child compilation.

Reply:
How is the child compiler affected by the its parent compilation mode?